### DISCUSSION TITLE
$__timeFilter in panel Title

### DISCUSSION THREAD

**Discussion by gustonator** [grafana/grafana#63238] [Dashboards, Panels & Visualizations]
Hi,
I would like to request a feature, to be able to use $timeFilter/ $__timeFilter in panel titles. 
I'm using Grafana 9.3.6 and it is still not working. 
According documentation, there is written you can use it in queries, which is great, but... why is it such a problem, to implement it to show also in panel titles? 
https://grafana.com/docs/grafana/latest/dashboards/variables/add-template-variables/#timefilter-or-__timefilter

I've tries multiple variations, but none of them is working
![image](https://user-images.githubusercontent.com/43645090/218042470-c3ea1c0e-f240-45f5-91ba-05f381454f54.png)

Many thanks
Gustonator

**Comment by i3laze** [comment]
My current workaround is a dashboard MySQL variable:
   `SELECT ($__to - $__from)/3600000`
Which gives me a label like "Last X hours".  

Watching this thread now :)

**Reply by gustonator** [reply]
@i3laze many thanks. As a workaroud, this works well. However you need to have MySQ DB datasource (which in my case I have - mysql + tsdb)

![image](https://user-images.githubusercontent.com/43645090/219322106-b5289298-8e0b-43c3-aa88-1f106b753989.png)

Still, I would be happy, if this would be build-in, because you cannot display texts like: "yesterday", "last week"...

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82821. Feel free to continue the discussion around this there. Thank you!
